Added documentation for #9733 (#181)
mautic/mautic-documentation: pages/22.users-and-roles/01.managing-users/docs.en.md

It is also possible to set the default timezone and language for the new user, and to set the user to published (able to log in) or unpublished (unable to log in).

> You need to provide the login credentials to the user to allow them to log in - ensure you do so in a secure way!

Reviewer: ```suggestion
You need to provide the login credentials to the user so that they can log in - ensure you do so in a secure way!
```

Reviewer: Fix exclamation mark for the Vale linter :)

Author: Done

> It's also possible to flag the option *Generate random password and send an email with credentials to user*, a password should be automatically generated and the user should receive an email with credentials.

Reviewer: ```suggestion
It is also possible to set the option *Generate random password and send an email with credentials to user*.  If this is selected, a password is automatically generated, and the user receives an email with their credentials.
```

Author: Done
